examples/calc/build.rs

```diff
@@ -0,0 +1,13 @@
+extern crate lrlex;
```

Comment:
Should this be builder.rs?

Reply:
No, `build.rs` is the file Cargo runs to build things.